src/analysis/analysis.cpp

```diff
@@ -6,10 +6,20 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   auto analysis_pkg = std::make_shared<StateDescriptor>("analysis");
   Params &params = analysis_pkg->AllParams();
   Real sigma = pin->GetOrAddReal("analysis", "sigma", 2e-3);
+  Real outside_pns_threshold =
+      pin->GetOrAddReal("analysis", "outside_pns_threshold",
+                        2.42e-5); // corresponds to entropy > 3 kb/baryon
+  Real inside_pns_threshold = pin->GetOrAddReal("analysis", "inside_pns_threshold",
+                                                0.008); // corresponds to r < 80 km
+  Real radius_cutoff_mdot =
+      pin->GetOrAddReal("analysis", "radius_cutoff_mdot", 0.04); // default 400km
```

Comment:
This stuff might better belong in the progenitor package.

Reply:
moved to progenitor pkg